theme-overrides/styles.css

```diff
@@ -1 +1,410 @@
+    margin: 0 !important;
+}
+
+.Tree-items>li::before {
+    content: " ";
+    width: 100%;
+    border-top: 1px solid var(--milestone-color-grey400);
+    position: absolute;
+}
+
+.Tree-items>li:first-of-type::before {
+    content: " ";
+    width: 100%;
+    border-top: 1px solid var(--milestone-color-grey400);
+    position: absolute;
+}
+
+.Tree-items>li:last-of-type::after {
+    content: " ";
+    width: 100%;
+    border-top: 1px solid var(--milestone-color-grey400);
+    position: absolute;
+}
+
+[dir="ltr"] .Tree-collection.is-closed .Tree-collectionLabel span::before {
+    background-image: none !important;
+    content: url('/src/demo-img/pil.svg');
+    opacity: 1 !important;
+    top: 3px !important;
+    left: 7px !important;
+    color: var(--milestone-color-primary-clear-blue);
+    transform: rotate(0deg) scale(.5);
+    transition: 150ms
+}
+
+[dir="ltr"] .Tree-collection.is-closed .Tree-collectionLabel {
+    transition: 150ms;
+    background: var(--milestone-color-white);
+}
+
+[dir="ltr"] .Tree-collection.is-closed .Tree-collectionLabel:hover {
+    transition: 150ms;
+    background: var(--milestone-color-grey100);
+}
+
+.Tree-collectionLabel span::before {
+    background-image: none !important;
+    content: url('/theme-overrides/images/pil.svg');
+    opacity: 1 !important;
+    top: 5px !important;
+    left: 8px !important;
+    color: var(--milestone-color-primary-clear-blue);
+    transform: rotate(90deg) scale(.5);
+    font-weight: 900;
+    transition: 150ms
+}
+
+.Tree-collectionLabel {
+    transition: 150ms;
+    background: var(--milestone-color-grey100);
+}
+
+.Tree-collectionLabel:hover {
+    background: var(--milestone-color-grey100);
+}
+
+.Tree-collectionLabel span {
+    font-weight: 400;
+    font-size: 13px !important;
+    color: var(--milestone-color-grey1000);
+    padding-left: 26px !important;
+}
+
+.Tree-entity a span {
+    color: #303436;
+    margin-left: 12px;
+}
+
+.is-current a {
+    background-color: #fff !important;
+}
+
+.is-current a span {
+    color: var(--milestone-color-primary-clear-blue) !important;
+    margin-left: 10px;
+}
+
+.Status-dot {
+    display: none;
+}
+
+.Tree-entityLink {
+    color: var(--milestone-color-grey1000);
+    font-weight: normal;
+    font-size: 12px;
+    padding-left: 30px !important;
+}
+
+.Tree-entityLink::before {
+    content: "";
+    width: 2px;
+    height: 16px;
+    background-color: var(--milestone-color-primary-clear-blue) !important;
+}
+
+.Tree-item .Tree-entityLink::before {
+    display: none;
+}
+
+.is-current .Tree-entityLink::before {
+    display: block;
+}
+
+.Tree-entityLink:hover {
+    background: unset !important;
+}
+
+.Tree-entityLink:hover span {
+    color: var(--milestone-color-primary-clear-blue);
+}
+
+.Frame-handle,
+.Preview-handle {
+    display: none;
+    width: 0 !important;
+}
+
+.Tree-collection.is-closed {
+    height: 50px;
+}
+
+h4.Tree-collectionLabel {
+    padding: 14px 0;
+}
+
+ol,
```

Comment:
please avoid styling html elements, if possible append a class and style that class